site/docs/es/demo/examples.md

```diff
@@ -0,0 +1,8 @@
+---
+prev: ./
+next: ./showlounge.md
```

Comment:
This line is not needed. https://raw.githubusercontent.com/grammyjs/website/main/site/docs/demo/examples.md

Reply:
Done